compiler/rustc_error_codes/src/error_codes/E0775.md

```diff
@@ -3,11 +3,10 @@ extension.
 
 Erroneous code example:
 
-```compile_fail,E0775
+```ignore (no longer emitted)
```

Comment:
Should this test also get the "Note: this error code is no longer emitted by the compiler." at the top?

Reply:
yes, I added that now. We had some thoughts about re-using this error code, but if we end up doing that we can just remove that note.